### PR TITLE
copier run, using previous answers

### PR DESCRIPTION
**Pull request by dougbrn**
This run is using the existing copier answers

### CODE REVIEW

File: .github/workflows/smoke-test.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    strategy:
+      matrix:
+        python-version: ['3.8', '3.9', '3.10']
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python {% raw %}${{ matrix.python-version }}{% endraw %}
+      uses: actions/setup-python@v4
+      with:
+        python-version: {% raw %}${{ matrix.python-version }}{% endraw %}
```
Comment (drewoldag) [inline .github/workflows/smoke-test.yml:23]: I just created this PR in the python project template to fix this: https://github.com/lincc-frameworks/python-project-template/pull/115/files

But basically, if you remove the `{% raw %}` and `{% endraw %}` tags on line 20 and 23, then this should become valid YAML, and the test should pass.

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -48,19 +56,20 @@ repos:
...
+    rev: 23.1.0
+    hooks:
+      - id: black
+        types: [python]
+        files: ^(src|tests)/
+        # It is recommended to specify the latest version of Python
+        # supported by your project here, or alternatively use
+        # pre-commit's default_language_version, see
+        # https://pre-commit.com/#top_level-default_language_version
+        language_version: python3.10
```
Comment (drewoldag) [inline .pre-commit-config.yaml:71]: nit: not sure if you want to update this to `python3.11`.

File: docs/conf.py [hunk 3]
```diff
@@ -8,52 +8,39 @@
...
 # -- Project information -----------------------------------------------------
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
 
-project = 'lsstseries'
-copyright = '2023, Doug Branton, Neven Caplar'
-author = 'Doug Branton, Neven Caplar'
-release = 'v0.1'
+project = "lsstseries"
+copyright = "2023, Doug Branton"
+author = "Doug Branton"
```
Comment (drewoldag) [inline docs/conf.py:21]: Looks like Neven's name was dropped on lines 20 and 21.

**Review by drewoldag — APPROVED** [review]
I left a couple small comments, but otherwise looks good to me.

File: docs/conf.py (hunk 3 above)
Comment (dougbrn) [reply docs/conf.py]: Changed to use the new lincc mailing list

**Pull request merged**